pkg/hive/metrics.go

```diff
@@ -59,6 +62,18 @@ func newMetrics() metrics {
 			Name:      "unreachable_peers_count",
 			Help:      "Number of peers that are unreachable.",
 		}),
+		PingTime: prometheus.NewHistogram(prometheus.HistogramOpts{
+			Namespace: m.Namespace,
+			Subsystem: subsystem,
+			Name:      "ping_time",
+			Help:      "The time spent for pings.",
```

Comment:
maybe worth mentioning that it expects ms. also, histogram needs buckets I think. it might be useful to define them in advance (not sure if it does automatic bucket division on its own)

Reply:
`DefBuckets = []float64{.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10}` default buckets if one is not specified.